libraries/core-react/CHANGELOG.md

```diff
@@ -5,6 +5,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.16.0] - 2021-12-02
+
+### Changed
+
+- Esm package is now unbundled enabling tree-shaking ([#1043](https://github.com/equinor/design-system/issues/1043))
+- Package.json: added `browserslist`: `defaults, not IE 11` resulting in less babel transpiled code and polyfill bloat ([#1043](https://github.com/equinor/design-system/issues/1043))
```

Comment:
Suggest removing "Package.json: added" and just say "Adjusted `browserslist`...  since we have just been using default browserlist before and that we added config to package.json seems to technical specific?